docs/appendices/release-notes/5.5.0.rst

```diff
@@ -81,7 +81,8 @@ SQL Statements
 SQL Standard and PostgreSQL Compatibility
 -----------------------------------------
 
-None
+- Allowed ``LIKE``/``ILIKE`` operators on ``INDEX OFF`` columns to return
```

Comment:
This should be removed, since we decided to treat as a bug fix, it will be backported, so no changes entry for 5.5.0 is required. Additionally it didn't throw an exception, it returned empty results:
```
cr> select * from t where b like '%o%';
+---+---+
| a | b |
+---+---+
+---+---+
SELECT 0 rows in set (0.011 sec)
```